stories/modal-dialog.stories.js

```diff
@@ -21,25 +21,23 @@
  */
 import { storiesOf } from '@storybook/react';
 
-/**
- * WordPress dependencies
- */
-import { __ } from '@wordpress/i18n';
-
 /**
  * Internal dependencies
  */
 import Dialog from '../assets/js/components/Dialog';
 
 storiesOf( 'Global', module )
 	.add( 'Modal Dialog', () => {
-		const { provides } = global._googlesitekitLegacyData.modules.analytics;
```

Comment:
The legacy data no longer has this key as it only exists in module registration now (JS) whereas before this was coming from PHP which has been removed for a while.